app/controllers/solidus_subscriptions/api/v1/subscriptions_controller.rb

```diff
@@ -4,7 +4,11 @@ class SolidusSubscriptions::Api::V1::SubscriptionsController < Spree::Api::BaseC
   def update
     if @subscription.update(subscription_params)
       persist_subscription_addresses(@subscription)
-      render json: @subscription.to_json(include: [:line_items, :shipping_address, :billing_address, :wallet_payment_source])
+      if params[:full_json]
+        render json: SubscriptionSerializer.new(@subscription).to_json(include: '**')
+      else
+        render json: @subscription.to_json(include: [:wallet_payment_source, :shipping_address, :billing_address])
```

Comment:
I note that this is a small behavior change from the old code, leaving out `:line_items` in the fallback case here.